Day-I-Social-Card/social-card/src/index.js

```diff
@@ -4,3 +4,8 @@ import './index.css';
 import App from './App';
 
 ReactDOM.render(<App />, document.getElementById('root'));
+
+const card = document.querySelector(".card-body");
+card.addEventListener('click', (e) => {
```

Comment:
I might ask you about this during stand up if I remember lol. I think I know what you are doing on this, but I want to double check